lib/jekyll/tags/highlight.rb

```diff
@@ -86,29 +70,10 @@ def parse_options(input)
         options
       end
 
-      def render_pygments(code, is_safe)
-        Jekyll::External.require_with_graceful_fail("pygments") unless defined?(Pygments)
-
-        highlighted_code = Pygments.highlight(
-          code,
-          :lexer   => @lang,
-          :options => sanitized_opts(@highlight_options, is_safe)
-        )
-
-        if highlighted_code.nil?
-          Jekyll.logger.error <<-MSG
-There was an error highlighting your code:
-
-#{code}
-
-While attempting to convert the above code, Pygments.rb returned an unacceptable value.
-This is usually a timeout problem solved by running `jekyll build` again.
-MSG
-          raise ArgumentError, "Pygments.rb returned an unacceptable value "\
-          "when attempting to highlight some code."
-        end
-
-        highlighted_code.sub('<div class="highlight"><pre>', "").sub("</pre></div>", "")
+      def render_pygments(code, _context)
+        Jekyll.logger.warn "Warning:",
```

Comment:
This should probably fail the build, right?

Otherwise, folks who use (ie) GitHub Pages will just stop having syntax highlighting in their code with no clue as to why, and that is _if_ they notice.

Reply:
yeah.. that's true..
I'd prefer users to keep abreast with changes in a major version and check for any warnings in their terminal output instead of getting a Page Build Error email from GitHub Pages..
but that's just me..

People just don't have the time to do a local run first.. :slightly_smiling_face: 
I'll update it to `raise` instead

Reply:
@pathawks Or... should we silently fallback to highlighting with `Rouge`? :bulb:

Reply:
@ashmaroli if there is no compatibility issue between the two highlighters, this could be the nicest solution for all users.

Reply:
From the README at `jneen/rouge`:
> Its HTML output is compatible with stylesheets designed for pygments.

If there's still a need for using `pygments` explicitly, the community can build a plugin which will be able to easily hook into the `render` method here.